pkg/internal/codegen/parse/crd.go

```diff
@@ -113,7 +113,13 @@ func (b *APIs) parseCRDs() {
 							resource.CRD.Spec.Subresources.Scale.LabelSelectorPath = &labelSelctor
 						}
 					}
-
+					if IsPrintColumn(resource.Type) {
+						result, err := parsePrintColumnParams(resource.Type)
+						if err != nil {
+							log.Fatalf("failed in parsing CRD, error: %v", err.Error())
```

Comment:
Update the error message to say "failed to parse printcolumn annotations,....." ?